src/explore-education-statistics-admin/src/pages/release/content/components/__tests__/EditableKeyStatText.test.tsx

```diff
@@ -167,4 +171,160 @@ describe('EditableKeyStatText', () => {
       screen.queryByRole('button', { name: /Remove/ }),
     ).not.toBeInTheDocument();
   });
+
+  describe('when editing', () => {
+    test('renders edit form correctly', async () => {
```

Comment:
This test seems wrong, it's testing the preview version, which is covered in other tests, rather than that clicking edit renders the form with populated inputs.

Reply:
Made similar changes to the test in `EditableKeyStatDataBlock.test.tsx`